tractor/_child.py

```diff
@@ -1,6 +1,53 @@
 import sys
 import trio
-import cloudpickle
+import argparse
+
+from ._actor import Actor
+from ._entry import _trio_main
+
+
+"""This is the "bootloader" for actors started using the native trio backend
+added in #128
+"""
+
 
 if __name__ == "__main__":
-    trio.run(cloudpickle.load(sys.stdin.buffer))
+
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("name")
```

Comment:
as discussed we should only need 3 args here:
- `uid`, `loglevel`, `parent_addr`

The rest can be sent at parent data transfer in `Actor._async_main()`.

Reply:
Also I think named args would be good, so like:
`uid=('name', '34lkj345-345lkj3450345')`, `loglevel='info'`, etc..